README.md

```diff
@@ -64,6 +64,70 @@ _Artwork by [Mike Sgier](http://msgierillustration.com)_
 <!-- No appveyor or gitter -->
 
 _This is the [Code.org](https://code.org) fork of johnny-five.  Please [check out the main project here!](https://github.com/rwaldron/johnny-five)_
+
+This project was forked in 2017 to provide better support for the [Adafruit Circuit Playground](https://learn.adafruit.com/introducing-circuit-playground/overview) which is used in the Code.org curriculum. The project was updated to the latest release of its upstream in September 2022, and Code.org customizations were then re-implemented.
+
+#### Source Code:
+
+``` bash
+git clone https://github.com/code-dot-org/johnny-five
+
+cd johnny-five
+
+npm install
+```
+
+## Building and testing with apps
+
+This is the most typical use case for @code-dot-org/johnny-five fork development.
+
+[Apps (aka Code Studio)](https://github.com/code-dot-org/code-dot-org/tree/staging/apps) is a set of apps which installs and references this package. You can run the dashboard locally from the apps directory using `bin/dashboard-server` ([setup reference](https://github.com/code-dot-org/code-dot-org/blob/staging/SETUP.md))
+
+Below are two options for local development and testing.
+
+1. Use [yarn link](https://classic.yarnpkg.com/en/docs/cli/link) 
+
+```
+cd {@code-dot-org/johnny-five repo directory}
+yarn link
+cd {code-dot-org repo directory}/apps
+yarn link @code-dot-org/johnny-five
+```
+
+When you are finished testing locally, you must undo yarn link in the reverse order:
+
+```
+cd {code-dot-org repo directory}/apps
+yarn unlink @code-dot-org/johnny-five
+cd {@code-dot-org/johnny-five repo directory}
+yarn unlink
+```
+
+The advantages of using `yarn link` is that changes in @code-dot-org/johnny-five are automatically transferred to @code-dot-org/code-dot-org/apps/node_modules and there are no local changes to package.json or yarn.lock. The disadvantage is that after you run the dashboard-server, you will need to use the Code.org Maker App locally to view and interact with the App lab with the enabled Maker Toolkit.
```

Comment:
> The disadvantage is that after you run the dashboard-server, you will need to use the Code.org Maker App locally to view and interact with the App lab with the enabled Maker Toolkit.

As discussed separately, it would be great to know why this is.

Reply:
The actual error you get when trying to connect with the Circuit Playground while on the localhost is 'Failure to open serial port.`
<img width="1140" alt="Screen Shot 2022-10-24 at 3 33 14 PM" src="https://user-images.githubusercontent.com/107423305/197652068-cda35ed5-c478-4d9f-87bb-1a19d9bb8b87.png">

Reply:
Per discussion with @breville offline, `yarn link` does not seem to work properly because johnny-five does not have a dist/ directory and does not build before being installed. I removed `yarn link` from the README because of its inconsistencies and resulting errors (example above).
I used `yarn add` for the code.org customizations and although it is a bit inconvenient, it works consistently.